presto-main/src/main/java/com/facebook/presto/operator/scalar/RegexpFunctions.java

```diff
@@ -195,16 +201,18 @@ private static void appendReplacement(SliceOutput result, Slice source, Regex pa
 
     @Description("string(s) extracted using the given pattern")
     @ScalarFunction
+    @LiteralParameters("x")
     @SqlType("array(varchar)")
```

Comment:
I think it is better to specify varchar(MAX_LENGTH) explicitly here. I don't know whether we are going to support unparametrized `varchar` signature for `varchar(x)` type.